### PR TITLE
Fixes #9654: rudder-dev typo

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9654

### CODE REVIEW

**Comment by peckpeck** [comment]
ping @amousset

**Comment by peckpeck** [comment]
PR rebased

File: scripts/rudder-dev/rudder-dev-src [hunk 1]
```diff
@@ -62,6 +63,24 @@ BRANCH
...
+    - merge if needed
+    If the type is fixed in a local file:
+    - check that the diff is small
+    - find the first branch having the problem
+    - open an issue of type bug with subject
+    - call rudder-dev branch on the issue
+    - patch the file
+    - commit with git
+    - call rudder dev commit
+    ex: rudder-dev typo README "Typo in readme"
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev-src]: change example to "rudder-dev quickfix"

File: scripts/rudder-dev/rudder-dev-src [hunk 2]
```diff
@@ -1105,6 +1132,50 @@ def wip():
   print("You can find the commit here: " + commit_url)
 
 
+# open a typo issue then and link it
+def quick_fix_pr(pr_url, trigraph=None, merge=False):
+  pr = PR(pr_url)
+  issue = Issue("9661")
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev-src]: Hardcoded issue ? :)

Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev-src]: overwritten below, you can remove it

File: scripts/rudder-dev/rudder-dev-src [hunk 3]
```diff
@@ -1105,6 +1132,50 @@ def wip():
...
+  # find PR
+  pr = PR(pr_url)
```
Comment (VinceMacBuche) [inline scripts/rudder-dev/rudder-dev-src]: two "pr = PR(pr_url)"

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**